generate-summary.py

```diff
@@ -13,7 +13,7 @@
 	total.append([0, list(tc)])
 	total[i][0] = i+1
 
-file = open(os.getcwd()+"\\assets\\data\\komunale.json", "rb")
+file = open("komunale.json", "rb")
```

Comment:
Now the path is wrong — just prepend it like I mentioned in the first comment today.